src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import '../App.css';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+const App = () => (
+  <div className="App">
+    <Display />
+    <ButtonPanel />
+  </div>
+);
```

Comment:
- [ ] Kindly wrap Display and ButtonPanel in React.Fragment helps to avoid adding an unnecessary DOM node.